### PR TITLE
LG-2616: Example Card Block

### PR DESCRIPTION
**Pull request by spark33**
image

+ success, info, warning variants

### CODE REVIEW

**Comment by audroue** [comment]
Looks good! (Went over fixes on local server.)

**Review by audroue — APPROVED** [review]

File: components/ContentstackRichText/ExampleCardBlock/ExampleCardBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const ExampleCardBlock = ({ entry }) => {
+  const IconComponent = Icons[entry.variant];
+  return (
+    <div>
+      <ImageContainer color={BorderColors[entry.variant]}>
+        <img src={entry.image.url} alt={entry.title} />
+      </ImageContainer>
+      <div
+        css={css`
```
Comment (brookescarlett) [inline components/ContentstackRichText/ExampleCardBlock/ExampleCardBlock.tsx:16]: can we define these outside of the function?

File: components/ContentstackRichText/ExampleCardBlock/ImageContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+import { palette } from '@leafygreen-ui/palette';
+
+// Same as AnnotatedImageBlock except for the color prop
```
Comment (brookescarlett) [inline components/ContentstackRichText/ExampleCardBlock/ImageContainer.tsx:4]: should we keep these styles in a shared location and inherit then?

Comment (spark33) [reply components/ContentstackRichText/ExampleCardBlock/ImageContainer.tsx]: They're not that similar any more actually - Rob and I made some tweaks :)

**Review by brookescarlett — APPROVED** [review]

**Pull request merged**